app/javascript/styles/components/Calendar.scss

```diff
@@ -89,3 +104,58 @@
     text-align: center;
   }
 }
+
+#jobs-year {
+  color: $color_font-medium;
+  font-family: $font_family-primary;
+  font-weight: bolder;
+  font-size: 60px;
+  margin-bottom: 1.5rem;
+}
+
+.jobs {
+  margin-right: 5rem;
+
+  text {
+    margin: 0;
+  }
+
+  &__title {
+    display: block;
+    font-family: $font_family-primary;
+    font-weight: bolder;
+    font-size: $font_size-large;
+    padding: 1rem 0;
+  }
+
+  &__subtitle {
+    display: block;
+    float: right;
+    color: $color_font-medium;
+    font-family: $font_family-primary;
+    font-weight: bold;
+    font-size: $font_size-medium;
+    left: -15rem;
+    margin-top: -8rem;
+    width: 17rem;
+  }
+
+  &__download-link {
+    display: block;
+    color: $color_font-medium;
+    cursor: pointer;
+    display: block;
+    font-family: $font_family-primary;
+    font-style: italic;
+    font-weight: 100;
+    margin-top: .5rem;
+    text-decoration: underline;
+  }
+
+  &__explanation {
+    display: block;
+    font-family: $font_family-secondary;
```

Comment:
The font family/size styling is inherited from our definition of paragraph styling, so you may want to remove those two lines here for DRYness.

Reply:
Done